### PR TITLE
[OSCD] suspicious csi.exe (rcsi.exe)  LOLBAS detection rule

### PR DESCRIPTION
**Pull request by grikos**
Tasks 117 & 128 from #1014

### CODE REVIEW

File: rules/windows/process_creation/win_susp_csi.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+title: Suspicious Csi.exe Usage 
+id: 40b95d31-1afc-469e-8d34-9a3a667d058e
+description: Csi.exe is a signed binary from Micosoft that comes with Visual Studio and provides C# interactive capabilities.
+  It can be used to run C# code from a file passed as a parameter in command line. Early version of this utility
+  provided with Microsoft “Roslyn” Community Technology Preview was named 'rcsi.exe'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_susp_csi.yml:5]: Hello @grikos! Could you please merge the description into a one line?

Comment (grikos) [reply rules/windows/process_creation/win_susp_csi.yml]: merged

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_susp_csi.yml]: Thank you very much!

**Comment by yugoslavskiy** [comment]
Thank you for your contribution, @grikos! Well done!

**Pull request merged**